framework/lib/components/search-bar/search-bar.tsx

```diff
@@ -97,7 +97,7 @@ export const SearchBar = forwardRef(
           defaultOptions={ filtered }
           loadOptions={ debouncedLoadOptions }
           onChange={ handleChange }
-          placeholder="Search ..."
```

Comment:
The changes you made to the file `search-bar.tsx` you should probably move to the `search-components-bar.tsx` which is specific to the search bar for the docs

Reply:
Fixed.